dist/js/better-jira.js

```diff
@@ -554,13 +554,16 @@ var Standup = function () {
       document.body.classList.add(this.cssClass);
 
       //-- Add Instructions element
-      if (document.querySelectorAll(this.instructionsCssClass).length === 0) {
+      if (document.querySelectorAll('.' + this.instructionsCssClass).length === 0) {
         var instructionsEl = document.createElement('div');
         instructionsEl.setAttribute('data-standup-close', '');
         instructionsEl.classList.add(this.instructionsCssClass);
         instructionsEl.innerHTML = '\n        <span class="text">Close Standup Mode <span class="close">&nbsp;&plus;&nbsp;</span></span>\n      ';
         document.body.appendChild(instructionsEl);
       }
+
+      //-- Hide breadcrumbs
+      document.querySelector('[data-testid="rapidboard-breadcrumbs"]').style.display = 'none';
```

Comment:
@brandonraydev One of the principles of this package is to only create the absolute bare minimum modifications to the _actual_ JIRA UI. This is for several reasons, the main one being compatibility and limiting possibility of JS errors that break the page. 

For example, in this case, if JIRA changes that testid (usually by removing the feature rather than modification, but it's still possible), turning on Stand Up mode would break at the very least this extension, at the most JIRA itself. 

Please move this modification into the SCSS.

Thanks!